### PR TITLE
Various combobox fixes

### PR DESCRIPTION
**Pull request by jpzwarte**

### CODE REVIEW

File: packages/components/combobox/src/combobox.ts [hunk 1]
```diff
@@ -503,6 +517,30 @@ export class Combobox<T = any, U = T> extends FormControlMixin(ScopedElementsMix
       }
     } else if (!this.wrapper?.matches(':popover-open') && ['ArrowDown', 'ArrowUp'].includes(event.key)) {
       this.wrapper?.showPopover();
+    } else if (['ArrowLeft', 'ArrowRight'].includes(event.key) && this.input.selectionStart === 0) {
```
Comment (anna-lach) [inline packages/components/combobox/src/combobox.ts:520]: In some cases I can get two tags focused at once (with arrow keys):

image

**Comment by Diaan** [comment]
I would remove the 'current' label from an option the moment the dropdown closes. Now you can close the dropdown with `esc` and then with enter you select an option you didn't even see was selected.

**Comment by Diaan** [comment]
https://github.com/user-attachments/assets/7d6c1d7f-5781-49cc-aa87-5e3acf9fd44f

This happens when i'm using only my up-arrow on the keyboard; the order of the options is a bit weird. Is it because they have the same index maybe?

File: .changeset/shiny-ants-cover.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+'@sl-design-system/combobox': patch
```
Comment (anna-lach) [inline .changeset/shiny-ants-cover.md:2]: Can you please also add changeset for other component's changes like grid, listbox, tag and so on? :)

**Review by anna-lach — APPROVED** [review]

**Comment by Diaan** [comment]
The tags in the lg variant are too big: 
image

**Pull request merged**